tests/simulator/test_advance.py

```diff
@@ -143,6 +168,258 @@ def vthz(x):
 
 
 
+    def _test_overlaped_fields_are_equal(self, time_step, time_step_nbr, datahier):
```

Comment:
that's a copy/paste from initialization test, do we want that?

Reply:
copy/paste/change
this one has advances, and I would say deprecates the initialization version, which I haven't removed but can as an addendum to this PR

Reply:
yes I would remove all initialization tests that are redundant with these advance tests

Reply:
commit/pushed/building